### PR TITLE
Sendsay api

### PR DESCRIPTION
**Pull request by T0R0NT0T0KY0**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
Снимок экрана 2023-05-28 в 9 28 19 PM

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://sendsay.ru/api/api.html>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/sendsay-api/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (weswigham) [inline types/sendsay-api/.eslintrc.json:1]: Delete this file and make the new types fir the project-wide lint settings, thanks~

**Review by weswigham — CHANGES_REQUESTED** [review]

File: types/sendsay-api/request-options.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export type SendsayRequestOptions = Record<string, any>;
```
Comment (weswigham) [inline types/sendsay-api/request-options.d.ts:1]: If this file doesn't exist at runtime (and it doesn't), just include these types in the file that uses them - same with the other type definition only files - thanks~

**Review by weswigham — CHANGES_REQUESTED** [review]

File: types/sendsay-api/.eslintrc.json (hunk 1 above)
Comment (T0R0NT0T0KY0) [reply types/sendsay-api/.eslintrc.json]: Thanks a lot!

Ref: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65625/commits/23d5ccaad4c839872695de028fa32d2e826dcea2

File: types/sendsay-api/request-options.d.ts (hunk 2 above)
Comment (T0R0NT0T0KY0) [reply types/sendsay-api/request-options.d.ts]: Thanks a lot!

Ref: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65625/commits/bda0ef3650c0ccf1102af075ab59eb68f6f6942c

**Review by sheetalkamat — APPROVED** [review]

**Pull request merged**